contrib/win32/win32compat/misc_internal.h

```diff
@@ -16,27 +16,26 @@
 #define IS_INVALID_HANDLE(h) ( ((NULL == h) || (INVALID_HANDLE_VALUE == h)) ? 1 : 0 )
 #define IS_VALID_HANDLE(h) (!IS_INVALID_HANDLE(h))
 
+#define errno_from_Win32LastError() errno_from_Win32Error(GetLastError())
+
+static char *machine_domain_name;
```

Comment:
did not find the usage of this variable

Reply:
Its an existing variable.. its used in getusergroups() in misc.c (openssh code)